### ISSUE TITLE
Deprecation warning of @SkipWhenEmpty since Gradle 7.4+

### ISSUE THREAD

**Issue by bernhardkern** [asciidoctor/asciidoctor-gradle-plugin#641]
## Versions:
`org.asciidoctor.jvm.convert` version `3.3.2`
`gradle` version `7.4.2`

## Issue

Running the Gradle task asciidoctor we receive the following deprecation warning, which was added to `Gradle 7.4+`

```
Relying on FileTrees for ignoring empty directories when using @SkipWhenEmpty has been deprecated. This is scheduled to be removed in Gradle 8.0. Annotate the property sourceFileTree with @IgnoreEmptyDirectories or remove @SkipWhenEmpty. Consult the upgrading guide for further information: https://docs.gradle.org/7.4.2/userguide/upgrading_version_7.html#empty_directories_file_tree
```

This is not related to issue https://github.com/asciidoctor/asciidoctor-gradle-plugin/issues/602, which deals with the mainClass deprection (Title would also match to this issue, but since it is a different topic, we used a more specific title.

**Comment by mrotteveel** [comment]
This seems to be a duplicate of #629

**Comment by ysb33r** [comment]
Fixed in 4.0.0-alpha.1
